src/test/resources/database-config.properties

```diff
@@ -0,0 +1,11 @@
+spring.datasource.url=jdbc:h2:mem:utopia;MODE=MYSQL;DATABASE_TO_UPPER=false
```

Comment:
I cannot remember. How did you supply h2 with a schema to run off of?

Reply:
I think Spring (Boot) automatically loads the contents of any `schema.sql` (only for tests if it's in the `src/test/resources` directory); the string `schema.sql` doesn't occur anywhere in the source code (including `pom.xml`) of our monolith.